public/campaigns/basic/assets/js/common.js

```
@@ -17,98 +13,195 @@ function scaleReviewPreviews() {
+
+function updateSliderSlideWidth(selector, slideWidth) {
+	document.querySelectorAll(selector).forEach(function(slider) {
+		slider.querySelectorAll(':scope > .slick-slide').forEach(function(slide) {
+			slide.style.flex = '0 0 ' + slideWidth;
+		});
+	});
+}
+
+/*============================================================
+スライド
+============================================================*/
```

ESLint

[error] Expected exception block, space or tab after '/*' in comment.

(spaced-comment)